### ISSUE TITLE
Cannot obtain a parameter value from a CLR object method

### ISSUE THREAD

**Issue by un-tone** [Starcounter/Starcounter.Linq#50]
Reported by @kegor.

```cs
private enum ItemType { Default, Custom }

// throws "System.NotSupportedException : Method call is not supported" on `.ToString()` method
Objects<Product>().Where(p => p.ItemType == ItemType.Default.ToString()).ToList();
```

In this case can be used `nameof(ItemType.Default)` which works well.
But the issue should be resolved for sure.

**Comment by un-tone** [comment]
The issue is resolved. NuGet packages are updated.

**Comment by un-tone** [comment]
The following case should be checked:
```cs
this.ChoosenTemplate = DbLinq.Objects<ProductTemplate>().FirstOrDefault(x => x.GetObjectNo() == DbHelper.Base64DecodeObjectID(this.ChoosenTemplateObjectId))
```

**Comment by un-tone** [comment]
Should work well. Re-opened by mistake (was tested on 2.0.4 while the issue was fixed in 2.0.5).
